README.md

```diff
@@ -234,5 +238,8 @@ To run without using Octopus configuration as code, include space_name and proje
         project_name: ${{ env.PROJECT_NAME }}
         version: ${{ steps.lazy-setup.outputs.image_version }}
         space_name: ${{ env.OCTOPUS_SPACE_NAME }}
+        charts_dir_path: charts
+      env:
+        IMAGE_NAME: ${{ steps.lazy-action.outputs.image_name}}
```

Comment:
Please don't do this. This means that everyone have to make the id of actions as lazy-actions? Why can't you export as env similar to how it is done in actions-setup?

Reply:
yes corrected. I realized it , by that time you added comment.